plugins/rapid7_insightvm/komand_rapid7_insightvm/util/util.py

```diff
@@ -91,3 +92,31 @@ def read_from_cache(filename):
         contents = cache_file.read()
 
         return contents
+
+
+def check_not_null(account: dict, var_name: str) -> Union[str, PluginException]:
+    """
+    Checks that a required value is inputted
+    :param account: user input
+    :param var_name: name of variable we check
+    :return: value or PluginException
+    """
+    value = account.get(var_name)
+    if value in (None, ""):
+        raise PluginException(cause=f"{var_name} has not been entered.", assistance=f"Enter valid {var_name}")
+    else:
+        return value
+
+
+def check_in_enum(value: str, var_name: str, possible_enums: list) -> None:
```

Comment:
```suggestion
def check_if_enum(value: str, var_name: str, possible_enums: list) -> None:
```
Also, if we're checking enums then do we really need to check enums if IC already only gives the user certain options?